src/plugins/data_source_management/public/components/direct_query_data_sources_components/connection_detail/direct_query_connection_detail.tsx

```diff
@@ -91,6 +95,24 @@ export const DirectQueryDataConnectionDetail: React.FC<DirectQueryDataConnection
     redirectToExplorerS3(dataSourceName, application);
   };
 
+  const dataSourceMenuView = useMemo(() => {
+    if (!featureFlagStatus) return null;
```

Comment:
what feature is this?

Reply:
That is the MDS feature flag. I didn't give a explicit name since the only flag in data source management was the mds. However, I think we should name it more explicitly. Maybe I can add that as a follow-up item.

Reply:
got it. explicit name would be better, people not familiar with the plugin won't understand what it means, and data source management might have other flags in the future